### DISCUSSION TITLE
👋 Who is using qwik-speak (showcase)?

### DISCUSSION THREAD

**Discussion by dejurin** [robisim74/qwik-speak#117] [Show and tell]
Since the usage of this library is growing, really curious which using are using qwik-speak. Very interesting to see the implementation of the library.
Leave a comment w/ **link website** or **github repo** here if qwik-speak is helpful to you! 🙂

**Comment by robisim74** [comment]
A list of projects hosted on Github that use the library can be found here: https://github.com/robisim74/qwik-speak/network/dependents

**Reply by dejurin** [reply]
With your permission, I would like to develop this topic here. For people to share finished projects on the prod. Something like a showcase.
I got the idea here: https://github.com/amannn/next-intl/issues/138

**Reply by robisim74** [reply]
No problem (and thank you for your time!): I pinned the discussion because I don't know how many dwell on the discussions

**Comment by dejurin** [comment]
I'll be the first

https://todaysdatenow.com/ - English
https://todaysdatenow.com/ro/ - Romanian
...

Subfolder style.

**Reply by robisim74** [reply]
Interesting and well done: I often need to compare dates and perform operations on them to manage complex processes: I usually use this site, especially for date operations tools: https://www.timeanddate.com/

But yours definitely has more languages!

**Reply by dejurin** [reply]
Honestly. This is my first project by qwik with qwik-speak. Half a year ago, my colleague told me about this wonderful framework, and I literally fell in love with this approach. And I decided to consolidate my knowledge by releasing this website. You have undoubtedly made a great contribution to development.
